.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+version: 2.1
+
+jobs:
+  build:
```

Comment:
I ran `circleci config validate` locally and I _think_ we need to include the [node orb](https://circleci.com/developer/orbs/orb/circleci/node#commands-install-packages). I got the error: `Cannot find orb 'node' looking for command named 'node/install-packages'`

Reply:
yeah im testing via circle ci now, not done here yet :)